policy/modules/kernel/files.fc

```diff
@@ -297,6 +297,7 @@ ifndef(`distro_redhat',`
 /var/run/.*			gen_context(system_u:object_r:var_run_t,s0)
 /var/run/.*\.*pid		<<none>>
 /var/run/lock/.*		<<none>>
+/var/run/motd.d(/.*)?		gen_context(system_u:object_r:etc_t,s0)
```

Comment:
It looks like this PR/commit is freely mixing `/var/run` and `/run`. Are the two *always* guaranteed to be symlinked? Given the original usecase, wouldn't it be better to use `/run` here?

Reply:
Yes, I just had a look at the FHS again. Makes sense to use `/run` as it is the newer one, and isn't a symlink - `/var/run` isn't guaranteed to be symlinked, I guess if it is an older system. Changing this, thanks!